boefjes/Makefile

```diff
@@ -43,7 +43,7 @@ images:  # Build the images for the containerized boefjes
 	# docker build -f images/base.Dockerfile -t openkat/dns-records --build-arg BOEFJE_PATH=./boefjes/plugins/kat_dns .
 	docker build -f ./boefjes/plugins/kat_dnssec/boefje.Dockerfile -t openkat/dns-sec .
 	docker build -f ./boefjes/plugins/kat_nmap_tcp/boefje.Dockerfile -t openkat/nmap  .
-
+	docker build -f ./boefjes/plugins/kat_hello_katty/boefje.Dockerfile -t openkat/hello-katty  .
```

Comment:
I believe the generic `images/base.Dockerfile` would suffice here, also see the commented line 43.

Reply:
I see! I thought `images/base.Dockerfile` was leftover from an older version of OpenKAT. I will use `images/base.Dockerfile` instead for my boefje